frontend/pages/main.css

```diff
@@ -36,4 +36,28 @@ body {
   background: #FFFFFF;
   border: 1px solid hsla(263, 71%, 65%, 1);
   color: hsla(263, 71%, 30%, 1);
+ }
+
+ a {
+   color:hsla(265, 64%, 50%, 1)
+ };
+
+ a:hover {
+   color:hsla(263, 71%, 39%, 1);
+ }
+
+ .flex {
```

Comment:
not so familiar with css and ant conventions. is this following a convention from a framework? if it is not, it might be neater to copy the ones we need from projects like tailwindcss as they have already thought of a comprehensive naming convention.

Reply:
yes this needs refactoring. probably a dedicated `module.css` which is what other components are doing